dev-packages/browser-integration-tests/utils/staticAssets.ts

```diff
@@ -22,23 +22,11 @@ export function addStaticAsset(localOutPath: string, fileName: string, cb: () =>
   symlinkAsset(newPath, path.join(localOutPath, fileName));
 }
 
-export function addStaticAssetSymlink(localOutPath: string, originalPath: string, fileName: string): void {
-  const newPath = path.join(STATIC_DIR, fileName);
-
-  // Only copy files once
-  if (!fs.existsSync(newPath)) {
-    fs.symlinkSync(originalPath, newPath);
-  }
-
-  symlinkAsset(newPath, path.join(localOutPath, fileName));
-}
-
-function symlinkAsset(originalPath: string, targetPath: string): void {
+export function symlinkAsset(originalPath: string, targetPath: string): void {
   try {
-    fs.unlinkSync(targetPath);
+    fs.linkSync(originalPath, targetPath);
   } catch {
-    // ignore errors here
+    // ignore errors here, probably means the file already exists
+    // Since we always build into a new directory for each test, we can safely ignore this
```

Comment:
Wondering if we are better off violently crashing, so that we catch this case when it happens...

Reply:
the problem is that this does happen :D tests fail without this. the alternative is to check if the file already exists before doing this, but a) this is in reality more overhead than try-catching this (because it only happens in a handful of tests where we have multiple HTML files), and b) it is still prone to flake because of possible race conditions. IMHO it is safe to do this, because each dir that it is building for is for a single test, where we never have the case/desire to have the same file being different from each other - they can safely be the same one always.